### PR TITLE
Update the bootstrap script to run a Django application

### LINKED ISSUE #8: Update bootstrap script to run a Django application
- Also update the script to run pipenv sync.
          This depends on a Pipfile.lock file that is created when installing Django (#3).

This task is dependent on Django having been installed on the Vagrant box (issue #3),
and on having a script to update (issue #7).

### PR DESCRIPTION
**Pull request by AmirMiller91**
The boostrap.sh script now installs Python dependencies and runs the Django server. 
After running "vagrant up" the website can be accessed locally
    (provided the Vagrant box doesn't already exist - to review this PR, run "vagrant destroy" first if it does).

.gitignore was updated accordingly.
Note: "db.sqlite3" was also added to .gitignore, since it is created by Django as part of the script.
          The task to add this change appears in the website part of the challenge (Part 5), 
          but I think it logically belongs in this PR. 

--- IMPORTANT ---
This PR is based on an unmerged branch (PR #13), that may be amended in the future.
It may require a rebase before it can be merged.

Closes #8.

### CODE REVIEW

**Review by maayanhd — COMMENTED** [review]
correct me if I'm wrong but after rebasing (after the PR linked to issue #13 will be merged, this PR will contain changes only at the bootstrap.sh file and the .gitignore file. If that is the case, then it looks fine to me, but the ultimate test will be testing the automation on each computer of the teammates after it will be merged to main. also, previous changes allowed you to not having in this PR changes in the Vagrantfile (the configuration of provisioning), again if I'm correct, then fine by me. I'll approve it after rebasing I believe.

**Review by Yarboa — COMMENTED** [review]
Please consider small chunks
asgi,wsgi are not a must here.

Try to follow this 
A working Vagrant environment that brings up the default django web server
A basic Django application that includes a static web page
A github repo in the Beyond organization with a README.md file that introduces your project
A basic Github workflow that executes Flake8 tests on files when there is a PR

**Comment by AmirMiller91** [comment]
Ma'ayan is are correct, this PR should only changes the bootstrap.sh and .gitignore files, since it is meant to resolve issue #8. It's a small task, that I believe was originally intended to be performed by the same contributor that installs Django on the Vagrant box. However, with the addition of a sixth team member, the Django and script tasks were split between two team members. I did not want to wait until PR #13 was merged to do do my task. Once it is merged, I can rebase this PR on the new main and force push - I don't expect any conflicts.

@maayanhd  - you don't have to wait for this PR to be merged or rebased if you want to check that it works. You can define my fork of the repository as a remote, fetch the branches, and checkout the branch "bootstrap_update_1". Then you can run "vagrant up" to test that it works.
I did not understand your last comment about the vagrantfile and provisioning - please clarify.

@Yarboa I'm sorry, I don't understand your comment.

**Comment by AmirMiller91** [comment]
OK, I see PR #13 was just merged. I will rebase and force push momentarily.

**Comment by maayanhd** [comment]
@AmirMiller91 I was referring to the PR linked to issue #7. I haven't had the chance to review it and remembered that logically it could have been incorporated in this PR. It was the part the configured the vagrant behavior of making the shell running the bootstrap file you made (which already happened again in the PR of the pipenv I mentioned), I'll wait for you to rebase and commit --amend and/or force push the changes, or whatever Yariv suggested on slack.

**Review by maayanhd — APPROVED** [review]
looking good, and working just fine

**Review by nadavspinco — APPROVED** [review]

**Review by shiraOs — APPROVED** [review]

**Review by Yarboa — APPROVED** [review]
Looks good

**Pull request merged**
